src/website/app/Table.js

```diff
@@ -20,7 +20,10 @@ import { createStyledComponent } from '../../utils';
 const styles = {
   cell: ({ theme }) => ({
     padding: `${theme.space_inset_md} ${theme.space_inset_sm}`,
-    verticalAlign: 'top',
+
+    '& > *': {
```

Comment:
See my comment on the `li` styles in Markdown.

Reply:
removed